distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -3506,6 +3519,8 @@ Once everything is configured and running, hooking up an existing {branding} ins
 
 The {branding} should now use the SSO/STS/LDAP servers when it attempts to authenticate a user upon an attempted log in.
 
+//  DDF-1796: Hardening Section Needs a lot of Clean up
```

Comment:
I update the ticket to say that this section needs to be removed. Please update the comment to reflect this.